docs/DeveloperGuide.md

```diff
@@ -663,14 +696,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Glossary
 
-* **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Applicant**: Represents a person that is searching for a job
 * **Applied Role**: Represents the job a person is applying for
 * **Employment Type**: Represents the type of work contract, specifically Full time, Part time, Temporary & Internship
 * **Expected Salary**: Represents the minimum amount the applicant is willing to be paid for the job
 * **Level of Education**: Represents the highest/current level of education the applicant has, specifically Elementary, Middle School, High School, University, Bachelors, Masters, PhD
-* **Years of Experience**: Represents the number of years the applicant has previously worked in their applied role for
+* **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Search term**: Terms that can be searched for. e.g. Search terms for roles are all the roles that exist in the list, meaning these are all the roles that can be searched for.
+* **Years of Experience**: Represents the number of years the applicant has previously worked in their applied role for
 --------------------------------------------------------------------------------------------------------------------
```

Comment:
I think that on top of these terms, we can add the definition of a `command` as it may not be so obvious to the user that it refers to a valid text input that executes certain features.
The term `command` is used in various places, particularly in implementation details:
![image](https://user-images.githubusercontent.com/61146198/140596761-46ef99e8-8405-4d8b-933b-0dd5960ad836.png)
![image](https://user-images.githubusercontent.com/61146198/140596777-123156f3-d968-4b11-8604-d325dcecc6c2.png)